### DISCUSSION TITLE
Notificaciones

### DISCUSSION THREAD

**Discussion by tecnicacearc** [riclolsen/OSHMI#6] [General]
Ricardo, buenas....nuevamente existe alguna actualizacion disponible para notificacion de eventos via mail/telegram?????

atte
Manuel

**Comment by riclolsen** [comment]
Dear Manuel.
There is no such module for OSHMI.
You can use Grafana alerting for this kind of integration.
You can also create a simple nodejs app using

https://www.npmjs.com/package/better-sqlite3 to read OSHMI sqlite tables, extract the data you need 

and

https://www.npmjs.com/package/telegram-notify to create telegram notifications.

Or use any other language you are comfortable with.

**Comment by tecnicacearc** [comment]
Gracias por la respuesta Ricardo...es posible con el plugins de Grafana
SQLite...levantar la base de datos hist.sql3... y las restantes????

Atte

Manuel

**Reply by riclolsen** [reply]
I have never tried, but it should work.
Use this plugin
https://grafana.com/grafana/plugins/frser-sqlite-datasource/
